simulators/history/portal-interop/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 
 [dependencies]
 ethportal-api = { git = "https://github.com/ethereum/trin", rev = "2a32224e3c2b0b80bc37c1b692c33016371f197a" }
```

Comment:
Why is `portal-hive` tagged to a particular hash of `ethportal-api`? If possible I think it's much more preferable to tag to the version

Reply:
Because ethportal-api has broken in the past and has broken our builds.

Sure that would be a fine change to make in an upcoming  PR

Reply:
I will make an issue for it